docs/advanced/peering/inter-cluster-network.md

```diff
@@ -26,9 +26,11 @@ The unpeer process will automatically remove the Liqo Gateway from the tenant na
 
 ## Manual on cluster couple
 
-When you have access to both clusters, you can configure the network connectivity for all the successive peering creations.
+When you have access to both clusters, you can configure the inter-cluster network connectivity via the `liqoctl network` command.
 
-First, you need to initialize the network:
+Note that when you use the `liqoctl network` command, the remote kubeconfig/context provided as argument, is the one of the cluster where we want to configure the server gateway of Wireguard tunnel.
+
+The first step to configure networking, is initializing the network configuration:
```

Comment:
Still missing what "initialize" means. What are we actually achieving with this command? Why is it needed?

Reply:
Fixed!
Initialize retrieves the network configurations of the clusters that needs to be connected, and creates all the resources to enable the IP remapping. The reason why this is a separate command is that this is a one shot action, but we are internally discussing whether it makes sense removing this command and making the `network init` implicit in the `network connect` command